Fix issue with table column resize in RTL (right to left) languages

bolah2009 opened the pull request:
<!--
Hi there - a lot of love for starting a Pull Request 😍. Please ensure the following things before creation - thank you!

- Create your Pull Request against the develop branch. We don't accept changes to stable branches.
- Add a reference to the issue you are trying to fix. Just add a # and the number.
- Don't run `bundle update`. It's a honorable thought of you but some dependency versions (e.g. pg) are broken 😢 Just use `bundle install` if you introduce new dependencies instead.
- Run `rubocop` and `coffeelint` on your changes to respect the code style guide.
- Add tests for your changes. Feel free to ask for support. We are happy to help you.

* The upper textblock will be removed automatically when you submit your Pull Request *
-->


# Description of issue

When language is set to an RTL language (for example, Hebrew), adjusting the column size of any table goes to the wrong direction, that is, when intending to reduce the size it increases and when intending to increase it reduces.

### Reference to issue #2977

# Proposed solution 

The `difference` variable in the `onColResizeMousemove` method is inverted (additive) for RTL languages, that is when the `html` have the `dir` attribute to be `rtl`, the `difference` variable is calculated by multiplying `-1` to it's calculated value.

# Testing

The following screenshot shows the changes

| Before changes | After Changes |
| :-----: | :-----: |
| ![rtl-zammad-bug](https://user-images.githubusercontent.com/36057474/77233104-8dbd7100-6ba5-11ea-9905-d563bdc70125.gif)  |![rtl-zammad-fixed](https://user-images.githubusercontent.com/36057474/77233107-90b86180-6ba5-11ea-94be-16c01502a50c.gif)  |

bolah2009 committed:
Fix issue with table column resize in RTL (right to left) languages

mantas commented:
Hi @bolah2009 ,

Thank you for you contribution.

I'd like to ask for a small change. The preferred way to detect text direction is `App.i18n.dir()` rather than checking DOM tree directly.

bolah2009 commented:
I was thinking if there is a better way too, I will do that. Thanks!

bolah2009 committed:
Make changes based on feedback

Use App.i18n.dir() instead of checking the DOM tree directly for text direction

mantas commented:
Thanks @ bolah2009 for the PR! This has been merged into develop with the commit above.

Unfortunately we're losing the reference as you as an author. That's why we add the PR and Github user reference.
